netdcli/netdcli.go

```diff
@@ -295,12 +295,13 @@ func validateOpts(opts *cliOpts) error {
 	if opts.pktTag == "auto" {
 		if opts.oper.Get() == CLI_OPER_CREATE &&
 			opts.construct.Get() == CLI_CONSTRUCT_NW {
-			log.Printf("  auto allocating network subnet from global pool")
+			log.Infof("  auto allocating network subnet from global pool")
 		}
 	} else if opts.pktTag != "" {
 		_, err = strconv.Atoi(opts.pktTag)
 		if err != nil {
 			log.Fatalf("Error convertinng tag %s to integer \n", opts.pktTag)
+			os.Exit(1)
```

Comment:
We may not need explicit os.Exit(1) after call to log.Fatalf() as I think Fatalf() will call exit.